src/features/game/expansion/components/ExpansionSuccess.tsx

```diff
@@ -0,0 +1,20 @@
+import { Button } from "components/ui/Button";
+import React, { useContext } from "react";
+
+import goblinJump from "src/assets/npcs/goblin_jump.gif";
+
+import { Context } from "features/game/GameProvider";
+
+export const ExpansionSuccess: React.FC = () => {
+  const { gameService } = useContext(Context);
+
+  return (
+    <div className="flex flex-col items-center">
+      <img src={goblinJump} className="w-1/2" />
```

Comment:
<img width="669" alt="Screen Shot 2022-07-26 at 4 59 34 pm" src="https://user-images.githubusercontent.com/11745561/180944106-a07aa924-3711-4bf3-b5a8-eeb48a67e42d.png">

The size of the goblin compared to the padding looks a bit off in this image. I would suggest using the Goblin + hammer (minting) gif that we have - it will fit the space a nicely 🤗